api_tests/nodes/views/test_node_alternative_citation_list.py

```diff
@@ -748,16 +748,21 @@ def test_add_citation_logged_out_private_reg(self):
         assert_equal(res.json['errors'][0]['detail'], 'Authentication credentials were not provided.')
         assert_equal(len(registration.alternative_citations), 0)
 
+
 class TestGetAlternativeCitations(ApiTestCase):
     def request(self, errors=False, is_admin=False, is_contrib=True, logged_out=False, **kwargs):
         admin = AuthUserFactory()
+        registration = kwargs.get('registration', None)
```

Comment:
Better to add registration to the function signature rather than pull of `kwargs`. Otherwise, the `registration` parameter gets passed to `create_project`, which doesn't really make sense. That said, it doesn't appear to break anything, so \shrug/